### PR TITLE
Form part of the project

### PR DESCRIPTION
**Pull request by VictorOmarT**
Task completed in this PR:

- Add  the form part of the project
- Learn how to  use the <form> tag
- Learn the hover and active mode

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -120,5 +120,32 @@ <h2 class="en">About me</h2>
...
+    <section class="form">
+      <form>
+        <h2 class="custom-font form-legend"> 
+          I'm always interested in hearing about new projects, so if you'd like to
+          chat please get in touch.
+        </h2>
+        <ul class="d-flex-colum">
+          <li>
+            <label for="name"></label>
+            <input class="input-text-email custom-font" type="text" id="name" placeholder="Full Name">
```
Comment (Shaher-11) [inline index.html]: The Name field accepts up to 30 chars. kidly fix that

File: index.html [hunk 2]
```diff
@@ -120,5 +120,32 @@ <h2 class="en">About me</h2>
...
+          </li>
+          <li>
+            <label for="mail"></label>
+            <input class="input-text-email custom-font" type="email" id="mail" placeholder="Email Adress" required>
+          </li>
+          <li class="text-area-container">
+            <label for="msg"></label>
+            <textarea class="textarea-message custom-font" name="message" id="msg" cols="30" rows="10" placeholder="Enter Text Here">
+            </textarea>
```
Comment (Shaher-11) [inline index.html]: The Message field accepts up to 500 chars.

File: index.html [hunk 3]
```diff
@@ -120,5 +120,32 @@ <h2 class="en">About me</h2>
...
+          <li>
+            <label for="name"></label>
+            <input class="input-text-email custom-font" type="text" id="name" placeholder="Full Name">
+          </li>
+          <li>
+            <label for="mail"></label>
+            <input class="input-text-email custom-font" type="email" id="mail" placeholder="Email Adress" required>
+          </li>
+          <li class="text-area-container">
+            <label for="msg"></label>
```
Comment (Shaher-11) [inline index.html:139]: The field needs a placeholder

File: index.html [hunk 4]
```diff
@@ -120,5 +120,32 @@ <h2 class="en">About me</h2>
...
+            <textarea class="textarea-message custom-font" name="message" id="msg" cols="30" rows="10" placeholder="Enter Text Here">
+            </textarea>
+            <img class="resizer-handle" src="./pictures/text_area/resizer_handle.png" alt="">
+          </li>
+          <li>
+            <button class="last-b custom-font" type="submit">Get in touch</button>
```
Comment (Shaher-11) [inline index.html:145]: The user cannot submit the form if one or more of the fields is empty, now I can submit the form with only the email
Kindly fix that

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @VictorOmarT,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Review by hemantso — APPROVED** [review]
Hi @VictorOmarT and  @nelsinofrancisco , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**